lib/src/widgets/new_recipe/item_new_gallery.dart

```diff
@@ -0,0 +1,173 @@
+                              ),
+                            )
+                          ],
+                        );
+                      } else {
+                        return Image.file(File(imageGallerys[index].path),
+                            fit: BoxFit.cover);
+                      }
+                    },
+                    staggeredTileBuilder: (index) => isTablet
+                        ? StaggeredTile.count(
+                            (index % 8 == 0) ? 4 : 1, (index % 8 == 0) ? 2 : 1)
+                        : StaggeredTile.count((index % 4 == 0) ? 3 : 1,
+                            (index % 4 == 0) ? 1.5 : 1),
+                    mainAxisSpacing: 10,
+                    crossAxisSpacing: 10,
+                  ),
+                ),
+                InkWell(
+                  onTap: () {
+                    showModalBottomSheet(
+                      context: context,
+                      builder: ((builder) => BottomSheetPickImage(
+                            typeImage: ImageType.imageForGallery,
+                          )),
+                    );
+                  },
+                  child: Container(
+                    height: 50.h,
+                    width: double.infinity,
+                    padding: EdgeInsets.only(
+                        top: 5.h, bottom: 5.h, left: 15.w, right: 0),
+                    decoration: BoxDecoration(
+                        border: Border.all(
+                            color: NewRecipeScreenColor.borderButtonColor),
+                        borderRadius: BorderRadius.circular(10)),
+                    child: Row(
+                      children: [
+                        Padding(
+                          padding: EdgeInsets.only(right: 10.w),
+                          child: Icon(
+                            Icons.add_outlined,
+                            size: 20,
+                            color: AppColor.primaryBlack.withOpacity(0.5),
+                          ),
+                        ),
+                        Expanded(
+                          child: Text(NewRecipeText.hintGalleryText,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .bodyText1!
+                                  .copyWith(
+                                      color: AppColor.primaryBlack
+                                          .withOpacity(0.5),
+                                      fontSize: 17)),
+                        ),
+                      ],
+                    ),
+                  ),
+                )
+              ],
+            ),
+          ),
+        ),
+        listener: (context, state) {
+          switch (state.runtimeType) {
+            case NewRecipeAddImageGallerySuccess:
+              state as NewRecipeAddImageGallerySuccess;
+              setState(() {
+                imageGallerys.addAll(state.listFile);
```

Comment:
If you choose 1 image for the first time and 2 images for the second time, will it output 3 images or 5 images?